build/build/src/rust/parser.rs

```diff
@@ -69,7 +69,6 @@ pub async fn run_self_tests(repo_root: &RepoRoot) -> Result {
         .output_ok()
         .await?
         .into_stdout_string()?;
-    trace!("Generated test code:\n{tests_code}");
```

Comment:
I have removed this print as it was printing TONS of data into stdout - it is printing the whole file which is rather large.

I don't think this is useful at all when not debugging the parser. If we need to introduce it, we can always re-enable. But in such case I'd prefer writing to a file that can then be downloaded from the workflow artifacts instead.